### PR TITLE
parametrize invitation labels

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

File: openreview/conference/templates/paper_recruitment_process.py [hunk 1]
```diff
@@ -56,9 +59,20 @@ def process(client, note, invitation):
 
 OpenReview Team'''
             response = client.post_message(subject, [edge.tail], message)
+
+            ## Send email to inviter
+            subject=f'[{SHORT_PHRASE}] {REVIEWER_NAME} {preferred_name} accepted to review paper {submission.number}, conflict detection pending'
+            message =f'''Hi {{{{fullname}}}},
+The {REVIEWER_NAME} {preferred_name} that you invited to review paper {submission.number} has accepted the invitation to review paper {submission.number}.
```
Comment (melisabok) [inline openreview/conference/templates/paper_recruitment_process.py:66]: @celestemartinez please check the wording here

File: openreview/conference/templates/paper_recruitment_process.py [hunk 2]
```diff
@@ -102,6 +116,16 @@ def process(client, note, invitation):
             ## - Send email
             response = client.post_message(subject, [edge.tail], message)
 
+            ## Send email to inviter
+            subject=f'[{SHORT_PHRASE}] {REVIEWER_NAME} {preferred_name} accepted to review paper {submission.number}'
+            message =f'''Hi {{{{fullname}}}},
```
Comment (melisabok) [inline openreview/conference/templates/paper_recruitment_process.py:121]: same here @celestemartinez

File: openreview/conference/templates/paper_recruitment_process.py [hunk 3]
```diff
@@ -130,5 +154,16 @@ def process(client, note, invitation):
         ## - Send email
         response = client.post_message(subject, [edge.tail], message)
 
+        ## Send email to inviter
+        subject=f'[{SHORT_PHRASE}] {REVIEWER_NAME} {preferred_name} declined to review paper {submission.number}'
```
Comment (melisabok) [inline openreview/conference/templates/paper_recruitment_process.py:158]: @celestemartinez same here

File: openreview/conference/templates/recruit_reviewers_pre_process.py [hunk 4]
```diff
@@ -7,9 +7,10 @@ def process(client, note, invitation):
     user = urllib.parse.unquote(note.content['user'])
     hashkey = HMAC.new(HASH_SEED.encode(), digestmod=SHA256).update(user.encode()).hexdigest()
 
+    if hashkey != note.content['key']:
```
Comment (melisabok) [inline openreview/conference/templates/recruit_reviewers_pre_process.py:10]: @celestemartinez I made a small change to throw the key error first, I'm worried that bots can try with different email addresses to guess who was invited.

File: openreview/conference/templates/paper_recruitment_process.py (hunk 1 above)
Comment (celestemartinez) [reply openreview/conference/templates/paper_recruitment_process.py:66]: Seems redundant to repeat the submission number twice. Maybe something like this:

> The reviewer {} that you invited to review paper {} has accepted the invitation.
> 
> Confirmation of the assignment is pending until the invited reviewer creates a profile in OpenReview and no conflicts of interest are detected.

File: openreview/conference/templates/paper_recruitment_process.py (hunk 2 above)
Comment (celestemartinez) [reply openreview/conference/templates/paper_recruitment_process.py:122]: It's trickier to not repeat the submission number here:
Option#1:

> The reviewer Melisa that you invited has accepted the invitation and is now assigned to paper 5.

Option#2:

> The reviewer Melisa that you invited to review paper 5 has accepted the invitation and is now assigned to paper 5.

Option#3:

> The reviewer you invited, Melisa, has accepted the invitation and is now assigned to paper 5.

what do you think?

Comment (melisabok) [reply openreview/conference/templates/paper_recruitment_process.py:128]: I like Option #2

**Pull request merged**